src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Entity.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@SuperBuilder
+@RequiredArgsConstructor
+@AllArgsConstructor
+@JsonInclude
+public abstract class Entity {
+    private String id;
+    private Identity identity;
+
+    private Name name;
+    private Address address;
+    private Contact contact;
+
+    private String organisation;
+
+    private YesNo confidentialityRequired;
+    private YesNo unacceptableCustomerBehaviour;
+    private YesNo vulnerableFlag;
+    private String vulnerabilityDetails;
+
+    private Subscription subscription;
+    private ReasonableAdjustmentDetails reasonableAdjustment;
```

Comment:
This is also a small one I think this should be renamed to resonableAdjustmentDetails just to make clear its the details as it sounds like it has YesNo resonableAdjustments

Reply:
that is its original name, id have to refactor more to change the name